sys-utils/setarch.c

```diff
@@ -82,6 +82,45 @@
 # define ADDR_LIMIT_3GB          0x8000000
 #endif
 
+#define ALL_PERSONALITIES \
+    X(PER_LINUX) \
+    X(PER_LINUX_32BIT) \
+    X(PER_LINUX_FDPIC) \
+    X(PER_SVR4) \
+    X(PER_SVR3) \
+    X(PER_SCOSVR3) \
+    X(PER_OSR5) \
+    X(PER_WYSEV386) \
+    X(PER_ISCR4) \
+    X(PER_BSD) \
+    X(PER_SUNOS) \
+    X(PER_XENIX) \
+    X(PER_LINUX32) \
+    X(PER_LINUX32_3GB) \
+    X(PER_IRIX32) \
+    X(PER_IRIXN32) \
+    X(PER_IRIX64) \
+    X(PER_RISCOS) \
+    X(PER_SOLARIS) \
+    X(PER_UW7) \
+    X(PER_OSF4) \
+    X(PER_HPUX) \
+    X(PER_MASK) \
+
+
+#define ALL_OPTIONS \
```

Comment:
On the man page, we call it "process personality flags"; what about ALL_PERSONALITY_FLAGS  ?

Reply:
I used the naming from `personality.h` for these lists.